### PR TITLE
Adding support for LZW60 Inovelli Device

### PR DESCRIPTION
**Pull request by matheus2308**
More details about product here: https://inovelli.com/4-in-1-sensor-z-wave/

### CODE REVIEW

File: config/inovelli/lzw60.xml [hunk 1]
```diff
@@ -0,0 +1,131 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/031E:000d:0001</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/inovelli/lzw31-sn.png</MetaDataItem>
```
Comment (Fishwaldo) [inline config/inovelli/lzw60.xml:4]: Eh. That doesn't seem like the right image.

File: config/inovelli/lzw60.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    <MetaDataItem id="0001" name="ZWProductPage" type="0001">https://products.z-wavealliance.org/products/3723</MetaDataItem>
```
Comment (Fishwaldo) [inline config/inovelli/lzw60.xml:5]: ID/Type don't match the database entry

File: config/inovelli/lzw60.xml [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    <MetaDataItem name="ProductPage">https://inovelli.com/shop/smart-sensors/z-wave-4-in-1-sensor/</MetaDataItem>
+    <MetaDataItem name="ProductSupport">https://inovelli.com/shop/smart-sensors/z-wave-4-in-1-sensor/</MetaDataItem>
+    <MetaDataItem name="ExclusionDescription">To Exclude your switch, put your HUB in exclusion mode and press the Sensor Button on the back of the device one (1) time.</MetaDataItem>
+    <MetaDataItem id="0001" name="FrequencyName" type="0001">U.S. / Canada / Mexico</MetaDataItem>
```
Comment (Fishwaldo) [inline config/inovelli/lzw60.xml:9]: Dito

File: config/inovelli/lzw60.xml (hunk 1 above)
Comment (matheus2308) [reply config/inovelli/lzw60.xml]: Ahh good point, I just added the correct one!

File: config/inovelli/lzw60.xml (hunk 2 above)
Comment (matheus2308) [reply config/inovelli/lzw60.xml]: ID and Type should be fixed now as well! Thank you.

**Pull request merged**